admin/BundleSolrBackup

```diff
@@ -0,0 +1,236 @@
+        next;
+    }
+    @extraneous_files = ();
+    for my $filename (sort grep { ! /^\.\.?$/ } readdir $bdh)
+    {
+        push @extraneous_files, $filename
+            if $filename ne $collection;
+    }
+    closedir $bdh;
+    die "$working_dir/$backup_dir has extraneous files: @extraneous_files\n"
+        if scalar @extraneous_files;
+    opendir(my $bsdh, "$working_dir/$backup_dir/$collection") or die $OS_ERROR;
+    for my $filename (sort grep { ! /^\.\.?$/ } readdir $bsdh)
+    {
+        push @extraneous_files, $filename
+            if ! $filename =~ 'backup_0\.properties|index|shard_backup_metadata|zk_backup';
+    }
+    closedir $bsdh;
+    die "$working_dir/$backup_dir/$collection has extraneous files: @extraneous_files\n"
+        if scalar @extraneous_files;
+
+    my $working_dump_dir = "$working_dir/$dump_dir";
+    move("$working_dir/$backup_dir", "$working_dump_dir") or die $OS_ERROR;
+
+    my $license_file = (grep { /^$collection$/ } @public_domain_collections)
+        ? "$FindBin::Bin/COPYING-PublicDomain"
+        : "$FindBin::Bin/COPYING-CCShareAlike";
+    copy($license_file, "$working_dump_dir/COPYING") or die $OS_ERROR;
+
+    my $readme_text = <<EOF;
+Backup timestamp: $BACKUP_STAMP
+
+The content of the subdirectory '$collection' is a backup of the search index by
+the same name from musicbrainz.org, in the new Apache Solr (8.9+) backup format.
+See https://solr.apache.org/guide/solr/latest/upgrade-notes/major-changes-in-solr-9.html#solr-8-9
+
+It is intended to be loaded in conjunction with an earlier MusicBrainz
+Postgres (full export) data dumps, preferably of the same day, in order
+to start a mirror with search kept in sync using the live data feed.
+See https://musicbrainz.org/doc/MusicBrainz_Server/Setup
+
+Since it has been created later than the above-mentioned full export,
+this backup has more updates, therefore these are NOT in sync
+and are NOT suitable for a snapshot/out-of-sync mirror.
+
+To load it, it is assumed that you use an up-to-date version of
+the MusicBrainz simple Solr search server schema (mbsssss)
+the MusicBrainz Solr (mb-solr) server in turn depends on.
+EOF
+    _write_file($working_dump_dir, 'README', $readme_text);
+
+    my $tar_file = "$collection.tar.zst";
+    _info("Creating $tar_file");
+    system
+        'bash', '-c',
+            'set -o pipefail; ' .
+            'tar' .
+            ' -C ' . shell_quote($working_dir) .
+            ' --create' .
+            ' --group=solr:8983' .
+            ' --owner=solr:8983' .
+            ' --remove-files' .
+            ' --verbose' .
+            ' -- ' .
+            shell_quote($dump_dir) .
+            " | zstd -T$COMPRESSION_THREADS -$COMPRESSION_LEVEL" .
+            ' > ' . shell_quote("$working_dir/$tar_file");
+    if ($CHILD_ERROR != 0)
+    {
+        _error('Tar returned ' . ($CHILD_ERROR >> 8));
+        exit 70; # EX_SOFTWARE
+    }
+
+    MusicBrainz::Script::MBDump::gpg_sign("$working_dir/$tar_file");
```

Comment:
`gpg_sign` should also work without the `MusicBrainz::Script::MBDump::` prefix since it was imported directly

Reply:
Correct. Now cleaned up through the commit https://github.com/metabrainz/musicbrainz-server/pull/3525/commits/95c35dbc7910b3ab2b41ae5584884e1285207569.